### ISSUE TITLE
How can i downgrade keras from 2.2.4 ?

### ISSUE THREAD

**Issue by anupamtamrakar** [keras-team/keras#12061]
With Tensorflow version 1.3 and Keras 2.2.4, i'm receiving error while creating sequential model

TypeError: softmax() got an unexpected keyword argument 'axis'

Help me to resolve this. 

One such resolution i can across is to downgrade keras to lower version as new version doesn't support axis argument in softmax(). What are the steps i need to perform to download it

**Comment by wwhio** [comment]
You can try `pip install keras==[version number]` to install a specific version of keras from pypi .
I browser github repo of tensorflow and notice that the date when the tensorflow 1.3 released is Aug 17, 2017. So i think you can install keras 2.1.2 which released on Dec 2, 2017 by github repo. The date is just a few months later than that of tensorflow. I don't verify this but i think it may work well.

TLDR, try this:

1. `pip uninstall keras`

2. `pip install keras==2.1.2`

**Comment by gabrieldemarmiesse** [comment]
Upgrading and downgrading python packages is unrelated to keras. This is more a question for stackoverflow. 
In your case, I would update tensorflow, it's *very* likely what causes the issue.

**Comment by P-Song** [comment]
Type the following command in colab will downgrade TensorFlow to version 1.15  and tf.keras to 2.2

%tensorflow_version 1.x

**Comment by SurendraReddy509** [comment]
%tensorflow_version 1.x .this command doen't work anymore because its updated
